Makefile

```diff
@@ -120,18 +114,25 @@ endif
 
 .PHONY: doc-setup
 doc-setup:
+	@go version > /dev/null 2>&1 || { echo "go is not installed for lxd-doc installation."; exit 1; }
```

Comment:
I think @tomponline meant to build the tool as part of the general Go build, not as part of the doc setup.

Reply:
Also, this here would mean that we build the tool twice when building the docs - once here in doc-setup and once during the actual doc build (conf.py).

We don't need this. But we want to see if the tool build breaks because of some code change, so it makes sense to add the tool build to the general build.